chapters/chapter2/3-random_forest.qmd

```diff
@@ -91,15 +106,16 @@ L'estimation **Out-of-Bag (OOB)** est une méthode particulièrement efficace po
 **Avantages de l'Estimation OOB**:
 
 - **Pas besoin de jeu de validation séparé** : L'un des principaux avantages de l'estimation OOB est qu'elle ne nécessite pas de réserver une partie des données pour la validation. Cela est particulièrement utile lorsque la taille du jeu de données est limitée, car toutes les données peuvent être utilisées pour l'entraînement tout en ayant une estimation fiable de la performance.
-- **Estimation directe et éfficace** : Contrairement à la validation croisée qui peut être coûteuse en temps de calcul, l'estimation OOB est disponible "gratuitement" pendant la construction des arbres. Cela permet d'évaluer la performance du modèle sans avoir besoin de réentraîner plusieurs fois le modèle.
+- **Estimation directe et efficace** : Contrairement à la validation croisée qui peut être coûteuse en temps de calcul, l'estimation OOB est disponible "gratuitement" pendant la construction des arbres. Cela permet d'évaluer la performance du modèle sans avoir besoin de réentraîner plusieurs fois le modèle et d'optimiser plus rapidement les hyperparamètres.
 - **Approximation de l'erreur de généralisation** : L'erreur OOB est considérée comme une bonne approximation de l'erreur de généralisation, comparable à celle obtenue par une validation croisée 10-fold. 
```

Comment:
Tu as une référence pour ça?

Reply:
Je pense que c'est Probst et al je vérifie!

Reply:
Probst et al 2019:

"3.2 Evaluation strategies and evaluation measures 

A typical strategy to evaluate the performance of an algorithm with different values of the hyperparameters in the context of tuning is k-fold cross-validation. The number k of folds is usually chosen between 2 and 10. Averaging the results of several repetitions of the whole cross-validation procedure provides more reliable results as the variance of the estimation is reduced (Seibold et al., 2018). In RF (or in general when bagging is used) another strategy is possible, namely using the out-of-bag observations to evaluate the trained algorithm. Generally, the results of this strategy are reliable (Breiman, 1996), i.e., approximate the performance of the RF on independent data reasonably well. A bias can be observed in special data situations (see Janitza and Hornung, 2018, and references therein), for example in very small datasets with n < 20, when there are many predictor variables and balanced classes. Since these problems are specific to particular and relatively rare situations and tuning based on out-of-bag-predictions has a much smaller runtime than procedures such as k-fold cross-validation (which is especially important in big datasets), we recommend the out-of-bag approach for tuning as an appropriate procedure for most datasets."